test/fetch/createFetchMock.ts

```diff
@@ -1,14 +1,16 @@
+import {FetchMock} from 'vitest-fetch-mock';
 import {MockedResponse} from '@Test/fetch/defineMockResponse';
 import {getAutoImplementation} from './getAutoImplementation';
-import {originalFetch} from './originalFetch';
 
-export const createFetchMock = <T>(implementation?: MockedResponse<T>, options?: Record<string, unknown>): jest.SpyInstance => {
-  originalFetch.set(global.fetch);
-
-  return jest.spyOn(global, 'fetch').mockImplementation(async (info: RequestInfo, init?: RequestInit) => {
+export const createFetchMock = <T>(
+  implementation?: MockedResponse<T>,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  options?: Record<string, unknown>,
+): FetchMock => {
+  return fetchMock.mockImplementation(async (info, init) => {
```

Comment:
deze fetchMock lijkt nergens vandaan te komen?

Reply:
uit hieronder test/mockFetch.ts zo blijkt? is dat een global standard?

Reply:
defined via setupFiles dus

Reply:
die is global en komt uit vitest-fetch-mock